mteb/tasks/Reranking/eng/BIRCODorisMaeInstructReranking.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskReranking import AbsTaskReranking
+
+
+class BIRCODorisMaeInstructReranking(AbsTaskReranking):
```

Comment:
Are there two versions of this dataset?  Why `InstructReranking` and `Reranking`?

Reply:
Given that some of these datasets are already in MTEB without the instruction (e.g. Arguana), I think we should only do the `instruction` version to avoid duduplication.